src/main/java/game/PlayerStats.java

```diff
@@ -0,0 +1,53 @@
+package game;
+
+public class PlayerStats extends Stats {
+
+    private int minHealthPoints;
+    private int maxHealthPoints;
+
```

Comment:
Is it not possible to combine the playerstats class with stats class? The stats are only for the player anyway right? You could have min, max and current healthpoints attribute in one class?

Reply:
The Stats class is also used to apply effects that change the PlayerStats. There is some confusion about this so we might change it up a bit